validator/src/main.rs

```diff
@@ -996,6 +996,18 @@ pub fn main() {
                 .help("The shred storage size in bytes. \
                        The suggested value is 50% of your ledger storage size in bytes."),
         )
+        .arg(
+            Arg::with_name("ledger_compression")
+                .hidden(true)
+                .long("ledger-compression")
```

Comment:
I think it's fair to prefix this arg by "rocksdb" as it is rocksdb-specific and there's a clear precedent for using this prefix with other similar configurations:
```suggestion
                .long("rocksdb-ledger-compression")
```